tools/inflector/inflector.go

```diff
@@ -6,18 +6,17 @@ import (
 	"unicode"
 )
 
-var columnifyRemoveRegex = regexp.MustCompile(`[^\w\.\*\-\_\@\#]+`)
-var snakecaseSplitRegex = regexp.MustCompile(`[\W_]+`)
+var (
+	columnifyRemoveRegex = regexp.MustCompile(`[^\w.*\-_@#]+`)
```

Comment:
This doesn't look like the same pattern. For example, `.` and `*` should remain unescaped because we want to preserve strings like `field.test` or `field[*]` (this is for a future feature for querying in json arrays).

Let's keep the original. If your change passes the tests, then probably we need to improve them.